src/tests/Interop/PInvoke/SafeHandles/SafeHandleTests.cs

```diff
@@ -16,14 +16,23 @@ public static void RunTest()
         {
             var testHandle = new TestSafeHandle(initialValue);
             Assert.IsTrue(SafeHandleNative.SafeHandleByValue(testHandle, initialValue));
+            Assert.IsTrue(SafeHandleNative.SafeHandleByValue(null, IntPtr.Zero));
```

Comment:
I believe that the PInvoke tests are not in the P0 set that runs on PRs by default. You may want to trigger them manually to make sure that they are passing.

Reply:
>I believe that the PInvoke tests are not in the P0 set that runs on PRs by default. 

That might explain why Mono didn't fail. I ran them locally on CoreCLR without issue. I will trigger them here for completeness.

Reply:
Did you run the test locally with Mono as well? It looks like the outerloop pipeline doesn't include a Mono build.

Reply:
>Did you run the test locally with Mono as well?

Ugh. This is infuriating. What the heck are we testing with all these pipelines.... sigh. I will try to set this up locally.

Reply:
Most of the outerloop pipelines that run the P1 tests are maintained by the CoreCLR JIT team, so they've probably never been updated to have a Mono build.

Reply:
Historically the way I've dealt with this for Mono is to run locally and then, when I want to validate on CI, temporarily make the test in question P0. If it passes everywhere, revert that and merge.

Reply:
I thought they were in the P0 tests?
The helix results seem to indicate the SafeHandleTests were run for both coreclr and mono.

Reply:
>Historically the way I've dealt with this for Mono is to run locally and then, when I want to validate on CI, temporarily make the test in question P0.

Yep, that is a completely reasonable work around. The only push back I have is that that is not a reasonable engineering system that one should expect to have to perform in order to be confident in ones work. I have been waiting hours for these CIs, now I am told none of the tests that would/should have validated any of this work in CI have been run. So I ask rhetorically - what have I been waiting for and why should I have any confidence in the CI?

I ran the test locally and it seems there is something amiss on Mono. Thanks for the suggestion to run locally.

Reply:
Amazing....

https://github.com/dotnet/runtime/blob/08be96571bddc04876d989c88129324be3433b3b/src/tests/Interop/PInvoke/SafeHandles/Program.cs#L15-L30

Reply:
I think we haven't had failing `SafeHandle` tests since https://github.com/dotnet/coreclr/pull/25784

Reply:
I am observing some other failures when running these tests against mono that appear unrelated to my changes here. I am updating the tests to fail properly which should enable us to determine if the SafeHandle tests are already failing and thus should be disabled for this specific test case. See https://github.com/dotnet/runtime/pull/48083

Reply:
I have verified the newly added marshaller support in Mono passes for the new `null` scenario with the SafeHandle tests. I have filed a tracking issue https://github.com/dotnet/runtime/issues/48084 and disabled the test on Mono with https://github.com/dotnet/runtime/pull/48083.